tests/solana_tests/constant.rs

```diff
@@ -24,8 +24,12 @@ fn constant() {
 
     let returns = vm.function("f", &[], &[], None);
     assert_eq!(returns, vec![Token::Uint(U256::from(42))]);
+}
 
-    let mut vm = build_solidity(
+#[test]
+#[should_panic(expected = "called `Option::unwrap()` on a `None` value")]
+fn constant_in_contract() {
+    build_solidity(
```

Comment:
Please use `fn parse_and_resolve()` for sema tests, and do an assert on the error.